lib/celluloid/groups/spawner.rb

```diff
@@ -19,9 +19,12 @@ def get(&block)
       end
 
       def each
-        threads = []
-        @mutex.synchronize { threads = @group.dup }
-        threads.each { |thread| yield thread }
+        to_a.each { |thread| yield thread }
+      end
+
+      # For temp compatiblity with specs
```

Comment:
How temporary is this, because I was definitely trying to avoid having this method.